docs/core/whats-new/dotnet-core-2-1.md

```diff
@@ -127,6 +127,9 @@ When you port existing code from the .NET Framework to .NET Core, you can use th
 
 .NET Core incorporates a new JIT compiler technology called *tiered compilation* (also known as *adaptive optimization*) that can significantly improve performance. Tiered compilation is an opt-in setting.
 
+> [!IMPORTANT]
+> Starting with .NET Core 2.2, tiered compilation is enabled by default.
```

Comment:
@richlander at one point I heard this was going to be off by default, where did that land?

Reply:
It is off by default, @Andrew-MSFT. I forgot that I'd added this to the 2.1 doc -- thanks for pointing it out.